src/qt/res/style.qss

```diff
@@ -424,18 +424,42 @@ QWidget#bgWidget {
     background: rgba(67, 138, 201, 235);
 }
 
-/** Ask passphrase Dialog **/
-QDialog#AskPassphraseDialog {
+/** Dialogs **/
+QDialog{
     border-radius: 5px;
     background-color: white;
 }
 
-QDialog#AskPassphraseDialog QLabel,
-QDialog#AskPassphraseDialog QCheckBox#toggleShowPasswordButton {
+QDialog QLabel,
```

Comment:
Can you keep them more specific ? you will impact some part of the send coin by doing so ;)

Reply:
yes sure